packages/host/tests/acceptance/code-submode/inspector-test.ts

```diff
@@ -850,7 +850,7 @@ module('Acceptance | code submode | inspector tests', function (hooks) {
   });
 
   test<TestContextWithSave>('can duplicate an instance in same realm', async function (assert) {
-    assert.expect(10);
+    assert.expect(7);
```

Comment:
an unnecessary save was being issued which doubled up the onSave assertions, fixed.